src/components/BookList.spec.js

```diff
@@ -34,92 +33,31 @@ describe('Book list', () => {
   ]
 
   it('renders two listitems', () => {
-    const mockOnHandleActiveReadingStatus = jest.fn()
-
-    render(
-      <BookList
-        books={books}
-        handleActiveReadingStatus={mockOnHandleActiveReadingStatus}
-      />
-    )
+    render(<BookList books={books} readingStatus={''} />)
 
     const bookListItems = screen.getAllByRole('listitem')
     expect(bookListItems).toHaveLength(2)
   })
 
-  it('has two buttons', () => {
-    const mockOnHandleActiveReadingStatus = jest.fn()
-
-    render(
-      <BookList
-        books={books}
-        handleActiveReadingStatus={mockOnHandleActiveReadingStatus}
-      />
-    )
-
-    const buttons = screen.getAllByRole('button')
-    expect(buttons).toHaveLength(2)
-  })
-
-  it('renders correct title according to readingStatus= "true"', () => {
-    const mockOnHandleActiveReadingStatus = jest.fn()
-
-    render(
-      <BookList
-        readingStatus={'true'}
-        books={books}
-        handleActiveReadingStatus={mockOnHandleActiveReadingStatus}
-      />
-    )
+  it('renders correct title according to readingStatus="finishedBooks"', () => {
+    render(<BookList books={books} readingStatus={'finishedBooks'} />)
 
     const title = screen.getByRole('heading', { level: 2 })
-    const buttons = screen.getAllByRole('button')
-    const finishedBooksButton = buttons[1]
     expect(title).toBeInTheDocument()
-    userEvent.click(finishedBooksButton)
     const titleFinishedBooks = screen.getByText(
       'Your library of finished books:'
     )
     expect(titleFinishedBooks).toBeInTheDocument()
```

Comment:
You can self yourself one `expect` by simply saying: `expect(title).toHaveTextContent('...)`. This tests both that the title is there and that it has the required text content.

Reply:
That is awesome! Changed it! :-)